### PR TITLE
More inactive cutoffs

### LINKED ISSUE #2827: [Feature request] More timestamps for idle feeds statistic page
I have some feeds that reactivate after a year or two so having only "last year" doesn't help me to find really idle feeds

So my proposal is also add "last 2 years", "last 3 years" and maybe even "last 5 years" if someone is even more patient than me

### PR DESCRIPTION
**Pull request by arokettu**
Closes #2827

Changes proposed in this pull request:

- Add more cutoff timestamps to the idle feeds stat page (2, 3, 5 years)

How to test the feature manually:

1. Have some idle feeds in your subscriptions
2. Go to idle feeds stat page
3. See additional date sections

Idle feeds that can be used to test:

more than 1 year: https://grrm.livejournal.com/data/rss
more than 2 years: http://joyreactor.com/rss/tag/pokemon+go
more than 3 years: http://drmcninja.com/feed/
more than 5 years: http://joyreactor.com/rss/tag/triple

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
  this part is not unit tested so I don't feel that I can write them from scratch
- [x] documentation updated
  this part does not seem to be documented

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

**Comment by arokettu** [comment]
One question: what to do with translations?

**Comment by Frenzie** [comment]
In the cli folder there's a translation helper script to auto-generate the entries.

File: app/Controllers/statsController.php [hunk 1]
```diff
@@ -73,6 +73,9 @@ public function indexAction() {
 	 *
 	 * It displays the list of idle feed for different period. The supported
 	 * periods are:
+	 *   - last 5 years
```
Comment (Frenzie) [inline app/Controllers/statsController.php:76]: Maybe also 10? 5 is peanuts, I've got tons of those.

(Then again, it's no more than a curiosity and I don't use that page.)

**Comment by arokettu** [comment]
Thank you. done

File: app/i18n/fr/gen.php [hunk 2]
```diff
@@ -73,7 +73,10 @@
 		'Jun' => '\\j\\u\\i\\n',
 		'jun' => 'juin',
 		'june' => 'juin',
+		'last_2_year' => 'Last two years',	// TODO - Translation
 		'last_3_month' => 'Depuis les trois derniers mois',
+		'last_3_year' => 'Last three years',	// TODO - Translation
+		'last_5_year' => 'Last five years',	// TODO - Translation
```
Comment (Alkarex) [inline app/i18n/fr/gen.php]:
```suggestion
		'last_2_year' => 'Depuis deux ans',
		'last_3_month' => 'Depuis les trois derniers mois',
		'last_3_year' => 'Depuis trois ans',
		'last_5_year' => 'Depuis cinq ans',
```

**Review by Alkarex — APPROVED** [review]

File: app/Controllers/statsController.php (hunk 1 above)
Comment (arokettu) [reply app/Controllers/statsController.php:76]: And I thought whether 5 is too much or not :D My goal was to add 3
I think if someone really needs a 10 years cutoff here, they can add a new pr for it

File: app/i18n/nl/gen.php [hunk 3]
```diff
@@ -73,7 +73,10 @@
 		'Jun' => '\\J\\u\\n\\i',
 		'jun' => 'jun',
 		'june' => 'Jun',
+		'last_2_year' => 'Last two years',	// TODO - Translation
```
Comment (Alkarex) [inline app/i18n/nl/gen.php]: @Frenzie Maybe you could suggest the translation?

Comment (Frenzie) [reply app/i18n/nl/gen.php]:
```suggestion
		'last_2_year' => 'Afgelopen twee jaar',
```

File: app/i18n/nl/gen.php [hunk 4]
```diff
@@ -73,7 +73,10 @@
...
 		'last_3_month' => 'Laatste drie maanden',
```
Comment (Frenzie) [inline app/i18n/nl/gen.php]:
```suggestion
		'last_3_month' => 'Afgelopen drie maanden',
```

File: app/i18n/nl/gen.php (hunk 3 above)
Comment (Frenzie) [reply app/i18n/nl/gen.php]:
```suggestion
		'last_2_year' => 'Afgelopen twee jaar',
```

File: app/i18n/nl/gen.php [hunk 5]
```diff
@@ -73,7 +73,10 @@
...
+		'last_3_year' => 'Last three years',	// TODO - Translation
```
Comment (Frenzie) [inline app/i18n/nl/gen.php]:
```suggestion
		'last_3_year' => 'Afgelopen drie jaar',
```

File: app/i18n/nl/gen.php [hunk 6]
```diff
@@ -73,7 +73,10 @@
...
+		'last_5_year' => 'Last five years',	// TODO - Translation
```
Comment (Frenzie) [inline app/i18n/nl/gen.php]:
```suggestion
		'last_5_year' => 'Afgelopen vijf jaar',
```

File: app/i18n/nl/gen.php [hunk 7]
```diff
@@ -73,7 +73,10 @@
...
 		'last_6_month' => 'Laatste zes maanden',
```
Comment (Frenzie) [inline app/i18n/nl/gen.php]:
```suggestion
		'last_6_month' => 'Afgelopen zes maanden',
```

**Pull request merged**